Gopkg.toml

```diff
@@ -0,0 +1,11 @@
+[[constraint]]
```

Comment:
Please switch the name and the branch, we are following the convention that deps start with name, then branch/commit.